controllers/AuthController.js

```diff
@@ -162,9 +162,12 @@ export default class AuthController {
         type,
       });
 
-      return successResponse(res, 200, 'user', { message: 'You have successfully logged in', token });
-    } catch (error) {
       /* istanbul ignore next-line */
+      req.token = token;
+
+      /* istanbul ignore next-line */
+      res.redirect(`${process.env.FRONTEND_URL}/signup?token=${token}`);
```

Comment:
Why am I seeing changes from the social login fix commit here? Try rebasing and then update the branch